spec/text_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../lib/text'
+
+describe Text do
+  subject { described_class.new(12345678) }
+  it { is_expected.to respond_to(:send_text) }
+
+  describe ".send_text" do
+    it "sends a message to given number" do
+      text = "Thank you! Your order was placed and will be delivered before #{(Time.now + 60 * 60).strftime("%k:%M")}"
```

Comment:
This expect is satisfied from within your test I think if you are using expect to receive it should be when this is called in another method.